src/ert/_c_wrappers/job_queue/ert_script.py

```diff
@@ -18,36 +20,18 @@ def __init__(self, ert: "EnKFMain"):
                 "ErtScript implementations must provide a method run(self, ert, ...)"
             )
         self.__ert = ert
-
-        self.__is_cancelled = False
-        self.__failed = False
```

Comment:
The commit message "Workflow state/status refactor" should be in imperative form "Refactor workflow state and status". It is also a bit big so if there is any way of splitting it that makes sense I would greatly appreciate it.

Reply:
I can rename the commit splitting it up into multiple logical parts is a bit difficult and it will not make it less big, but the refactor is not critical so I can just drop the commit.